flashcard.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/helper.rb'
+
+class FlashcardRunner
+  def initialize(round, deck, cards)
+    @round = round
+    @deck = deck
+    @cards = cards
+  end
+
+  def start
+    puts "Welcome! You're playing with 4 cards.
```

Comment:
It would be more dynamic to interpolate with something like `#{@round.deck.cards.count}` instead of hardcoding the number of cards here and on line 13.